### PR TITLE
Fixes fart noises

### LINKED ISSUE #20034: Farting power
(WEB REPORT BY: kyunny REMOTE: 172.93.109.202:7777)
When you have superfart genetic power and you try to *fart, you do normal and superfart at the same time, except only normal fart fires correctly while superfart makes no sound.

### PR DESCRIPTION
**Pull request by ShiftyRail**
Closes #20034 
Also makes the file less indent gore

Tested

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -74,130 +74,133 @@
...
-			var/list/farts = list(
-				"farts",
-				"passes wind",
-				"toots",
-				"farts [pick("lightly", "tenderly", "softly", "with care")]",
-			)
-
-			if(H.mind.miming)
-				farts = list("silently farts.", "acts out a fart.", "lets out a silent fart.")
+	if(H.op_stage.butt == 4)
```
Comment (jknpj) [inline code/modules/mob/living/carbon/human/emote.dm]: dare you enter my magical butt number

**Comment by DamianX** [comment]
unreviewable

**Comment by ShiftyRail** [comment]
>Break someting
>Fixes it
>Unreviewable

I'll fix the magic number even though it was already here before but for the rest I don't really know what I can do
Maybe try to see the diff in Split view or view directly the edited file ; all I did is to move some checks to do an early return if possible and changed a `get_turf(src)` (because src doesn't have a turf)

**Comment by ShiftyRail** [comment]
Farts are properly delayed : 
delay on farts
Butt surgeries work : 
surgery
Superfart works :
superfart
(I could hear the sounds)
And biblefart works : 
biblefart

File: __DEFINES/surgery_defines.dm [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Butt removal
+
+#define SURGERY_HAS_A_BUTT	   0
+#define SURGEY_BUTT_CUT		   1
```
Comment (DamianX) [inline __DEFINES/surgery_defines.dm]: SURGEY

**Pull request merged**